src/components/footnote-group/footnote-group.scss

```diff
@@ -0,0 +1,28 @@
+// TODO: Remove unused
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/focus';
+@use '../../mixins/ms';
+
+/// Footnote groups are used to group footnotes together in a list
+.c-footnote-group {
+  --spacing-footnotes: #{ms.step(2)};
+
+  border-top: size.$edge-medium solid color.$base-gray-lighter;
```

Comment:
Can we use a darker color for the dark theme? The border feels much more prevalent there than it does in the light theme.

Reply:
Oh yeah, great catch :+1:

Reply:
I changed the border color. I also changed the `:target` color for dark themes. Is this working better? Would you change anything?
![Uploading Screen Shot 2021-09-02 at 1.44.33 PM.png…]()

Reply:
@Paul-Hebert Could we make the border color the same as the one used for the comment thread lines instead of a new, lighter blue like it is here?